substrate/frame/revive/dev-node/runtime/src/lib.rs

```diff
@@ -356,7 +356,7 @@ pallet_revive::impl_runtime_apis_plus_revive_traits!(
 			VERSION
 		}
 
-		fn execute_block(block: Block) {
+		fn execute_block(block: <Block as polkadot_sdk::polkadot_sdk_frame::traits::Block>::LazyBlock) {
```

Comment:
`polkadot_sdk_frame` has some import clause above where you can add `traits::Block`.

Reply:
Nice. Done.